.env.example

```diff
@@ -10,6 +10,22 @@ CUBEJS_DB_TYPE=postgres
 CUBEJS_API_SECRET= # random 32-digit base64 string - generate from https://generate.plus/en/base64
 CUBEJS_DEV_MODE=true
 
+PORT=4000
+
 # local web server
-APP_API_PORT=4000
 APP_DASHBOARD_PORT=3000
+
+# redis configuration
+CUBEJS_REDIS_URL='redis://host.docker.internal:6379'
```

Comment:
for local - this is not even needed right?

Reply:
yeah redis is not needed for local setup. Cubejs switches to memory when dev mode is enabled.